src/raygun.utilities.js

```diff
@@ -127,7 +127,7 @@ var raygunUtilityFactory = function (window) {
       },
       
       isReactNative: function () {
-          return typeof document === 'undefined' && '__DEV__' in window;
+          return typeof document === 'undefined' && typeof window['__DEV__'] !== 'undefined';
       },
```

Comment:
@fundead Unfortunately I also tried this and it didn't work. Or at least, the release bundle gets created fine but it seems to return false, so I get the launch error about document not existing.